scripts/scene_library.gd

```diff
@@ -1356,6 +1362,8 @@ func _on_item_list_item_clicked(index: int, at_position: Vector2, mouse_button_i
 	if selected_assets.size() == 1: # If only one asset is selected.
 		popup.add_item("Open", AssetContextMenu.OPEN_ASSET)
 		popup.set_item_icon(-1, get_theme_icon(&"Load", &"EditorIcons"))
+		popup.add_item("Inherit", AssetContextMenu.INHERIT_ASSET)
```

Comment:
```suggestion
		popup.add_item("New Inherited Scene", AssetContextMenu.INHERIT_ASSET)
```

Reply:
Should `popup.add_item("Open", AssetContextMenu.OPEN_ASSET)` also say Open Scene then to match the FileSystem options?

Reply:
> Should `popup.add_item("Open", AssetContextMenu.OPEN_ASSET)` also say Open Scene then to match the FileSystem options?

I think yes.